src/components/LinterMessage/styles.module.scss

```diff
@@ -1,3 +1,43 @@
+$top-arrow-width: 10px;
+$top-arrow-height: 20px;
+
+.inline:first-child {
+  &:after,
+  &:before {
+    border-color: transparent;
+    border-style: solid;
+    content: '';
+    display: block;
+    height: 0;
+    left: 10px;
```

Comment:
Could this be `$top-arrow-width`? It's the same value.

Reply:
I did not use this variable because this is the number of pixels to put the arrow slightly on the right (from the left border).